source/app/views/gameboard.erb

```diff
@@ -0,0 +1,57 @@
+<div class="row">
+        <div class="col-md-12">
+                <form action='/game' class="form-horizontal" method="post">
+                  <input type='hidden' name='player_1_id' value='<%= user1_id %>' />
+                  <input type='hidden' name='player_2_id' value='<%= user2_id %>' />
+                        <legend class="text-center header"><h1>Play "The Rock", Parchment, Fiskars</h1></legend>
+                        <div id="player1" class="form-group">
+                           <h3>Player 1</h3>
+                           <div class="row">
+                            <div class='col-md-4'>
+                              <div class="radio">
+                              <label><input type="radio" name="optradio"><img src="/images/rock.png"></label>
```

Comment:
Same thing with these input radio buttons, we have a token table for a reason! We can just iterate through the tokens and make options for them!